source/api_documentation/network_integration/ringpools/index.rst

```diff
@@ -85,6 +85,12 @@ Endpoint:
    :description: Delete a RingPool
    :page: delete_ring_pool
 
+.. api_endpoint::
+   :verb: DELETE
+   :path: /ring_pools/&lt;customer_phone_number&gt;
+   :description: Delete a CustomerPhoneNumber
+   :page: customer_phone_number
```

Comment:
Path and page is incorrect here. Also, we are calling them Destinations in documentation, unless we want to change it to Customer Phone Number.

```suggestion
   :path: /customer_phone_numbers/&lt;customer_phone_number_id&gt;
   :description: Delete a Destination
   :page: delete_customer_phone_number
```